### PR TITLE
fix: Increase UI readability by using relative sizes and comply to WCAG

### PR DESCRIPTION
**Pull request by Wachiwi**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [x] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

- [x] I changed the sizing variables to utilize rem on the basis of https://css-tricks.com/accessible-font-sizing-explained/ so that fonts get better readable on larger resolutions/displays.
- [x] I added special variables for the font size since it does not use the same scaling as the other sizes (e.g. default font-size should not be _tiny_). All usages of `size.*` for font-sizes where adapted to use the new variable `fontSize.*`.
- [x] I aligned the usage of the sizing variable for fonts to comply with the WCAG guidelines. Commonly used font size is 16+ with some exceptions, e.g. e.g in the issue list, where a visual hierarchy is needed

## Related Tickets & Documents

Fixes: #621

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

TBD

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [x] Would love to add accessibility tests e.g. with cypress, but perhaps would be a separate PR

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by Wachiwi** [comment]
One open discussion point aka question from my side would be, if all fonts really should be at least 16px or there is still one with a size of 14px for some information (e.g. side-notes)

### 16px (smallest) => 18px default
Screenshot 2020-11-01 at 19 36 11
Screenshot 2020-11-01 at 19 36 23

### 14px (smallest) => 16px default

Screenshot 2020-11-01 at 19 36 45
Screenshot 2020-11-01 at 19 36 38

File: src/styles/variables.js [hunk 1]
```diff
@@ -21,11 +21,11 @@ const colors = {
 };
 
 const margin = {
-  gutter: "10px",
+  gutter: "1rem",
 };
 
 const padding = {
-  sides: "30px",
+  sides: "3rem",
```
Comment (bdougie) [inline src/styles/variables.js:28]: I don't think changing the global margin and padding is within scope. This is managed with media tags and meant for spacing and not font-sizes.

File: src/styles/variables.js [hunk 2]
```diff
@@ -38,16 +38,16 @@ const typography = {
...
+  tiny: "1.6rem",
+  small: "2.4rem",
+  medium: "4rem",
+  large: "6.4rem",
+  xl: "10.4rem",
+  xxl: "16.8rem",
 };
 
-const borderRadius = "5px";
+const borderRadius = "0.5rem";
```
Comment (bdougie) [inline src/styles/variables.js:50]: Keep this at px

File: src/index.css [hunk 3]
```diff
@@ -1,76 +1,49 @@
...
+  --large: 6.4rem;
+  --xl: 10.4rem;
+  --xxl: 16.8rem;
 
   /* border radius */
-  --smallBorderRadius: 4px;
-
-  --borderRadius: 10px;
+  --smallBorderRadius: 0.4rem;
+  --borderRadius: 1rem;
```
Comment (bdougie) [inline src/index.css:40]: px for radius

**Review by bdougie — CHANGES_REQUESTED** [review]
Please keep spacing and radius variables to px and not rems

**Comment by bdougie** [comment]
> One open discussion point aka question from my side would be, if all fonts really should be at least 16px or there is still one with a size of 14px for some information (e.g. side-notes)

I don't think 16px should be the goal. Your use of rems is fine since the user can zoom their browser and the proportions will all still match.

**Comment by bdougie** [comment]
I took a look and saw that the H1 is smaller. Can you adjust this?

**before**: 
Screen Shot 2020-11-11 at 5 11 02 AM

**after**:
Screen Shot 2020-11-11 at 5 10 06 AM

**before**:
Screen Shot 2020-11-11 at 5 10 23 AM

**after:**
Screen Shot 2020-11-11 at 5 23 16 AM

File: src/styles/Card.js [hunk 4]
```diff
@@ -126,7 +126,7 @@ const ContextStyle = styled(Card)`
 
   h1 {
     width: 100%;
-    font-size: ${fontSize.xl};
+    font-size: ${fontSize.xxl};
```
Comment (bdougie) [inline src/styles/Card.js:129]: LGTM

**Review by bdougie — APPROVED** [review]

**Comment by bdougie** [comment]
Looks good to merge. @Wachiwi if you could update the package-lock.json to fix the conflicts, I can merge.

**Comment by Wachiwi** [comment]
Updated package-lock.json

**Review by filiptronicek — APPROVED** [review]

**Pull request merged**